[2022-07-28 10:21:17,143] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: user_processing.process_user manual__2022-07-28T10:21:11.212618+00:00 [queued]>
[2022-07-28 10:21:17,151] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: user_processing.process_user manual__2022-07-28T10:21:11.212618+00:00 [queued]>
[2022-07-28 10:21:17,151] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-07-28 10:21:17,152] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-07-28 10:21:17,152] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-07-28 10:21:17,166] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): process_user> on 2022-07-28 10:21:11.212618+00:00
[2022-07-28 10:21:17,170] {standard_task_runner.py:52} INFO - Started process 1304 to run task
[2022-07-28 10:21:17,173] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'process_user', 'manual__2022-07-28T10:21:11.212618+00:00', '--job-id', '39', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmpmxyg8xu3', '--error-file', '/tmp/tmpqwkwwj42']
[2022-07-28 10:21:17,174] {standard_task_runner.py:80} INFO - Job 39: Subtask process_user
[2022-07-28 10:21:17,230] {task_command.py:369} INFO - Running <TaskInstance: user_processing.process_user manual__2022-07-28T10:21:11.212618+00:00 [running]> on host 890323d6f4a0
[2022-07-28 10:21:17,297] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=process_user
AIRFLOW_CTX_EXECUTION_DATE=2022-07-28T10:21:11.212618+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-07-28T10:21:11.212618+00:00
[2022-07-28 10:21:17,312] {python.py:173} INFO - Done. Returned value was: None
[2022-07-28 10:21:17,323] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=process_user, execution_date=20220728T102111, start_date=20220728T102117, end_date=20220728T102117
[2022-07-28 10:21:17,346] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-28 10:21:17,412] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
